Add files via upload

JoePapage opened the pull request:
# Your checklist for this pull request

## About your game

What is your game about? 
Pac-Man is trying to collect all the stars needed for his star collection but some spooky ghosts stand in his way.

How to play your game?
Press WASD to move.

## Code
Check off the items that are true.
- [ Y] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [Y ] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [Y ] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [ Y] The game runs without errors. 
- [Y ] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [Y ] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [ ] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ ] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

> Thanks for PR!

LucasHT22 commented:
Wow, pacman, I loved it! but we need to solve this bug:

`Uncaught TypeError: Cannot read properties of undefined (reading 'y') on line 487 in column 4. Open the browser console for more information.`

If you need help, I'm here:)

JoePapage commented:
> Wow, pacman, I loved it! but we need to solve this bug:
> 
> `Uncaught TypeError: Cannot read properties of undefined (reading 'y') on line 487 in column 4. Open the browser console for more information.`
> 
> If you need help, I'm here:)
> 

Hi, thanks for the feedback. So, I fixed the bug, and added a new input, j, to restart the game. Do I add another file or...

LucasHT22 commented:
No! `Files changed` > `...` > `edit`

JoePapage commented:
Hey, I've finished all of the changes and reviewed the code, everything should be good now!

LucasHT22 reviewed (approved):
Great changes! Looks good to me, Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

out of curiosity, how did you meet sprig?

JoePapage commented:
Thanks for all the help! I met sprig through the HackClub at my highschool.

LucasHT22 commented:
oh, cool!
